### DISCUSSION TITLE
Generating an svg from Floorplanner

### DISCUSSION THREAD

**Discussion by jtermi03** [ExperienceLovelace/ha-floorplan#183] [Q&A] [answered]
I must have gotten mixed up between the various guides on how to approach this, but after spending a day creating my floorplan using [Floorplanner](floorplanner.com/), I found that they only allow export in either png or their proprietary fml format.  Am I missing something here? 

Also, to clarify something basic how this all is supposed to behave, because it hasn't been totally clear in the documentation, is the following correct?

1. If I start with a png floorplan, I convert it to svg and then use ha-floorplan to overlay png's (also converted to svg) of the room with lights on when the state of the entity changes to on
2. If I start with an svg floorplan, ha-floorplan can modify the light entity in the svg to dynamically 'turn on' the lighting in that room

Meaning that it's definitely worth my time to start fresh?

**Comment by exetico** [comment, marked as answer]
Hi

It's mentioned that the Floorplanner-picture are only used as an background image, if you follow through the guide. Else I must remember something wrong.

Take a look at this discussion:
https://github.com/ExperienceLovelace/ha-floorplan/discussions/131#discussioncomment-1654167

I hope that will help you. 

I understand that there's still a lot of space for Improvements in the docs. But I've not found the proper amount of time to push changes to it ☺️.

**Reply by jtermi03** [reply]
Oh I'm certain the mistake was on my end, I'm pretty fresh to all of this, specifically around how svgs are manipulated/leveraged. 

To confirm then, are those 'extended' capabilities to dynamically light possible if I start fresh with something in svg?

**Reply by OzGav** [reply]
Have a look at this thread and this post in particular and see if it clarifies things?
https://community.home-assistant.io/t/floorplan-now-available-as-a-lovelace-card/115489/416?u=ozgav

**Reply by jtermi03** [reply]
@OzGav that explanation was excellent, it was exactly what I needed as someone with a patchwork of technical understanding. I think my confusion wasn't with the guides, but with the impressiveness of the tool - I interpreted the fading in and out of "lights on" svgs as dynamic lighting changes, but this makes much more sense! Thanks!

**Reply by OzGav** [reply]
Excellent. It would be great if you could mark this as answered and then if you have any more questions you can search the discussions or open a new thread. Happy New Year!
